recipes/uvw/all/test_package/test_package.cpp

```diff
@@ -1,8 +1,6 @@
-#include <cassert>
-
 #include <uvw.hpp>
 
 int main() {
     auto loop = uvw::Loop::getDefault();
-    assert(loop != nullptr);
+    loop->run();
```

Comment:
This won't run anything because there are no handlers registered in the event loop.

I didn't like the original test because `assert` is a macro that will get compiled out in a release configuration.

Let me know if I should drop the line entirely then.

Reply:
Same, let me discuss it in the office with @uilianries tomorrow and I'll get back to you!

Reply:
OK. Will update.